misc/python/materialize/cloudtest/k8s/testdrive.py

```diff
@@ -49,6 +49,7 @@ def run(
         input: Optional[str] = None,
         no_reset: bool = False,
         seed: Optional[int] = None,
+        default_timeout: int = 300,
```

Comment:
To match with the behavior of default_timeout in mzcompose/services.py, can you make this an `Optional[str]` with a default of `'300s'`, that is, include the unit in the value rather than tacking `s` implicitly.